content/posts/daily-driver-ebike.md

```diff
@@ -0,0 +1,105 @@
+---
```

Comment:
@KevMca if you could make sure I didn't get anything wrong in this that would be mighty 😁